buildtest/utils/file.py

```diff
@@ -12,7 +12,7 @@
 import logging
 
 from buildtest.defaults import logID
-from buildtest.log import BuildTestError
+from buildtest.utils.command import BuildTestError
```

Comment:
hmm, should this be under `buildtest.utils.exceptions`?  Or actually, where I usually expect custom exceptions for a library would be is under `buildtest.exceptions` directly. Do you consider this a command?

Reply:
Sure i can move this over to ``exceptions.py`` give me a second to refactor the code

Reply:
Awesome! And I think it's most intuitive to find at the root (right under buildtest) if that works for you too. I can't tell you how many times I've been writing something to catch an error, and interactively I look for some `<module>.exceptions` to import from.